Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml

```diff
@@ -0,0 +1,1268 @@
+    type: regular
+    task:
+      id: 344e1330-e5f8-4292-83df-7a4fba147ebf
+      version: -1
+      name: Get host information from ExtraHop Reveal(x)
+      description: Get host information from ExtraHop Reveal(x).
+      script: '|||extrahop-device-search'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      ip:
+        complex:
+          root: inputs.IPAddress
+      name:
+        complex:
+          root: inputs.Hostname
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 2100,
+          "y": 890
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+    continueonerrortype: ""
+  "33":
+    id: "33"
+    taskid: cb25c554-4bce-4f19-8bf1-974a6a51c57d
+    type: condition
+    task:
+      id: cb25c554-4bce-4f19-8bf1-974a6a51c57d
+      version: -1
+      name: is Mcafee ePolicy Orchestrator v2 enabled
+      description: "Gets a value and returns it. This is to be used in playbook conditional tasks - get a value from incident field, label or context, and act accordingly. \nIf an array is returned. the first value will be the decision making value."
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      "yes":
+      - "34"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: modules
+                filters:
+                - - operator: isExists
+                    left:
+                      value:
+                        simple: modules.brand
+                      iscontext: true
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: modules.state
+                      iscontext: true
+                    right:
+                      value:
+                        simple: active
+                accessor: brand
+            iscontext: true
+          right:
+            value:
+              simple: McAfee ePO v2
+    view: |-
+      {
+        "position": {
+          "x": 420,
+          "y": 710
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+    continueonerrortype: ""
+  "34":
+    id: "34"
+    taskid: 968db0d2-282a-4a9b-8071-39de4a0b5b09
+    type: regular
+    task:
+      id: 968db0d2-282a-4a9b-8071-39de4a0b5b09
+      version: -1
+      name: Get- host information from McAfee ePO v2
+      description: Finds systems in the McAfee ePO system tree.
+      script: McAfee ePO v2|||epo-find-system
```

Comment:
Use 'epo-find-system ' instead of 'epo-find-system (McAfee ePO v2)'

Reply:
Done.